### PR TITLE
Add UI to gauge quadrant

### PR DESCRIPTION
**Pull request by matraj**
Another branch that needs to be merged. This one adds the UI to the gauges quadrant.

image

### CODE REVIEW

File: src/styles/Accelerometer.css [hunk 1]
```diff
@@ -1,13 +1,15 @@
 .Accelerometer-combined {
-  display: flex;
-  align-items: center;
+/*  display: flex;
```
Comment (rageandqq) [inline src/styles/Accelerometer.css]: Remove commented code.

File: src/styles/Gauge.css [hunk 2]
```diff
@@ -1,24 +1,52 @@
...
   height: 100%;
   width: 100%;
-  display: flex;
-  align-items: center;
+  /*align-items: center;*/
+  padding: 1px 16px;
 }
 
-.Gauge-left,
+.Gauge-left {
```
Comment (rageandqq) [inline src/styles/Gauge.css]: Remove the whole style if its unused.

File: src/styles/Gauge.css [hunk 3]
```diff
@@ -1,24 +1,52 @@
...
+
+.Gauge-card {
+  background: linear-gradient(to bottom, #202023 0%, #101011 100%);
+  border-radius: 5px;
+  height: 90%;
+  width: 100%;
+  padding: 1px 1px;
 }
 
 .Gauge-values {
```
Comment (rageandqq) [inline src/styles/Gauge.css]: Same here.

File: src/styles/Gauge.css [hunk 4]
```diff
@@ -1,24 +1,52 @@
...
+  font-size: 48px;
+  font-style: normal;
+  font-variant: normal;
+  font-weight: 100;
+  line-height: 26.4px;
 }
 
-.Gauage-value {
-  padding: 20px;
+.Gauge-accel {
```
Comment (rageandqq) [inline src/styles/Gauge.css]: And here.

File: src/styles/Gauge.css (hunk 3 above)
Comment (matraj) [reply src/styles/Gauge.css]: Done

File: src/styles/Gauge.css (hunk 4 above)
Comment (matraj) [reply src/styles/Gauge.css]: Done

**Pull request merged**